addons/mail/models/mail_thread.py

```diff
@@ -2135,6 +2135,51 @@ def _message_create(self, values_list):
     # NOTIFICATION API
     # ------------------------------------------------------
 
+    def _notify_cancel_by_type_std(self, notification_type):
```

Comment:
``_notify_cancel_by_type_sql`` ? or ``_notify_cancel_by_type_generic`` ? Just that "std" makes me think it is Stephane's work :D .